canonical/snapcraft.io
# fix: Fix form state bug on listing page

## Done
Fixed a bug where the save and revert buttons weren't disabled once a change had been reverted

## How to QA
- Go to https://snapcraft-io-4736.demos.haus/<snap_name>/listing
- Check that you can change fields, change them back to their original value and the "Save" and "Revert" buttons go back to being disabled
  - It is known that it doesn't work for the licences field due to its complexity

## Testing

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-12782
